src/main/resources/MessagingDescriptor.json

```
@@ -0,0 +1,28 @@
+{
+  "publications": [
+    {
+      "eventType": "ITEM_CHECKED_OUT",
+      "description": "Item check out event",
+      "eventTTL": 1,
+      "signed": false
+    },
+    {
+      "eventType": "ITEM_CHECKED_IN",
+      "description": "Item check in event",
+      "eventTTL": 1,
+      "signed": false
+    },
+    {
+      "eventType": "ITEM_DECLARED_LOST",
+      "description": "Item declared lost event",
+      "eventTTL": 1,
+      "signed": false
+    },
+    {
+      "eventType": "LOAN_DUE_DATE_UPDATE",
```

Could this be phrased in the past tense?

Does this represent any change in the due date, for any reason, or specifically a manual change to due date?

```suggestion
      "eventType": "LOAN_DUE_DATE_UPDATED",
```

---

The associated test uses the language `due date changed` is that a more appropriate domain phrase for this event?

---

Done